### ISSUE TITLE
Modify the Search input box style.

### ISSUE THREAD

**Issue by MohitSuhane** [Imaginea/KodeBeagle#515]
Currently if we try to search with long keywords or multiple keywords whose size is more than the Search input box size, then the keyword gets overlap with the remove and search image.

We need to expand the input box in case the keyword size is more.
Please see below image for issue.

![inputboxissue](https://cloud.githubusercontent.com/assets/5663528/16007125/4c41c0aa-318e-11e6-857d-d919bf63c256.png)

**Comment by hashd** [comment]
While this is an issue, this is probably highly unlikely when an end user uses. 

Instead of increasing the text box size, we could have an horizontal scrollbar maybe?

**Comment by MohitSuhane** [comment]
Yes including a horizontal scroll bar may be a good option because the same input box is there on the second page as well where we show the results of the search.
